subsys/spm/spm_ns.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <logging/log_ctrl.h>
+#include <logging/log.h>
+#include <spm.h>
+#include <init.h>
+
+LOG_MODULE_REGISTER(spm_ns);
+
+static void log_dump(void)
+{
+	LOG_PANIC();
+}
+
+static int spm_ns_init(const struct device *dev)
+{
+	ARG_UNUSED(dev);
+	int err = spm_ns_set_fatal_error_handler(log_dump);
```

Comment:
Where is the implementation of spm_ns_set_fatal_error_handler()? I can't find it in the PR, sorry if I'm missing something :P.

Reply:
that is a mistery :) somehow it got lost. Fixed now.